scripts/python/createDataWithFaker.py

```diff
@@ -0,0 +1,22 @@
+"""
+Checkout Faker and create Fake Data for future use.
+"""
+import json
+
+from faker import Faker
+
+fake = Faker()
+
+def createYourListe():
+    kunde = []
+    for i in range(1, 11):
+        i = fake.profile()
+        kunde.append(i)
+
+    print(kunde)
```

Comment:
Hatten wir nicht gesagt das du das als newline Json machen sollst ? machmal bitte nicht nur print sondern speicher das in ner json datei, dann wirst du leichter sehen ob es valid ist oder nicht.
Und wolltest du nicht die Struktur nachbauen ?